src/PseudoPromise.ts

```diff
@@ -9,7 +9,7 @@ type CatcherItem = { type: 'catcher'; catcher: Catcher };
 type Items = (FuncItem | CatcherItem)[];
 
 export class PseudoPromise<ReturnType = undefined> {
-  readonly _return: ReturnType;
+  readonly _return: ReturnType | undefined;
```

Comment:
Why was this changed?

Reply:
I get the following error when undefined is not added

````
error: TS2564 [ERROR]: Property '_return' has no initializer and is not definitely assigned in the constructor.
  readonly _return: ReturnType;
````